### PR TITLE
minor changes.

### PR DESCRIPTION
**Pull request by kartikeyakirar**
part of https://github.com/insightsengineering/teal.transform/pull/176

I have made minor updates as part of review process
- updated document
- changed tags 
- added some punctuations
- removed unused variable.

### CODE REVIEW

File: R/choices_labeled.R [hunk 1]
```diff
@@ -374,11 +379,7 @@ value_choices.data.frame <- function(data,
   res
 }
 
-#' Print choices_labeled object
-#'
-#' `r lifecycle::badge("stable")`
-#'
-#' @rdname choices_labeled
+#' @describeIn choices_labeled Print choices_labeled object
```
Comment (averissimo) [inline R/choices_labeled.R:382]: The `@return` tag needs some unifying, I suggest 1 of these (my preference with first):

1. Don't have a `@return` tag for the `print` function. It's a very common function that other packages don't write about the return statement
2. `@return` tag prefixed with "#' @return `print` returns..."
3. Remove the `@return` tag and add what it returns to the `@describeIn` text.

image

File: R/choices_selected.R [hunk 2]
```diff
@@ -34,8 +34,8 @@ no_select_keyword <- "-- no selection --"
 #' be on top of the drop-down field.
 #' @param fixed (optional `logical`) Whether to block user to select choices.
 #'
-#' @return `choices_selected` returns object of class `choices_selected` and of
-#' type list which contains the specified `choices`, `selected`, `keep_order` and `fixed`.
+#' @return `choices_selected` list object, encapsulating the specified
```
Comment (averissimo) [inline R/choices_selected.R:37]: It's repetitive, but given it appears alongside `is.choices_selected, it seems necessary. (if you go with 3. of my other comment, then apply it here too)

```suggestion
#' @return `choices_selected` returns list of `choices_selected`, encapsulating the specified
```

image

File: R/get_merge_call.R [hunk 3]
```diff
@@ -3,7 +3,7 @@
 #' @description
 #' `r lifecycle::badge("stable")`
 #'
-#' Returns list of calls depending on selector(s) and type of the merge.
+#' Function creates list of calls depending on selector(s) and type of the merge.
```
Comment (averissimo) [inline R/get_merge_call.R:6]:
```suggestion
#' Creates list of calls depending on selector(s) and type of the merge.
```

File: R/include_css_js.R [hunk 4]
```diff
@@ -2,8 +2,8 @@
 #'
 #' `system.file` should not be used to access files in other packages, it does
```
Comment (averissimo) [inline R/include_css_js.R:3]: I'd add this as a `@note` given the content, but it's ok if it stays like this

File: R/merge_expression_module.R [hunk 5]
```diff
@@ -1,5 +1,6 @@
 #' Merge expression module
 #'
+#' @description
 #' `r lifecycle::badge("experimental")`
 #'
 #' This function is a convenient wrapper to combine `data_extract_multiple_srv()` and
```
Comment (averissimo) [inline R/merge_expression_module.R:6]: This was previously part of `@details` hence the structure

I'd remove the start of the phrase.

```suggestion
#' Convenient wrapper to combine `data_extract_multiple_srv()` and
```

**Review by averissimo — COMMENTED** [review]
Great job, left some comments that may require a bit of discussion.

Do you want to huddle for a fast resolution?

File: R/choices_labeled.R (hunk 1 above)
Comment (kartikeyakirar) [reply R/choices_labeled.R:382]: removed return tag here [891271c](https://github.com/insightsengineering/teal.transform/pull/197/commits/891271c1f6a30f47a40e1251ad6f9d3e2cc76de1)

**Review by averissimo — APPROVED** [review]
LGTM

**Pull request merged**